libs/cardano-ledger-core/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## 1.15.0.0
 
+* Add `member'` function to `UMap` module. #4639
+  * It checks for membership directly against `umElems` instead of a `UView`.
```

Comment:
That should be a haddock, not an entry in changelog.
Changelogs should only contain useful information about what changed, not explain how the new functionality behaves. Behavior is for documentation
The only time when we need to add comments like that when we make semantic changes to some function, especially if that function's type signature did not change  (which is a scenario we should avoid if at all possible).
```suggestion
```